docs/reference/versions/public.md

```diff
@@ -1,28 +1,22 @@
-# Public Cloud
```

Comment:
Don't remove this heading. (This is what `pymarkdown` is complaining about.)

Reply:
Got it, thanks!